daffodil-runtime1/src/main/scala/org/apache/daffodil/runtime1/processors/PackedDecimalParsers.scala

```diff
@@ -84,11 +84,10 @@ class PackedDecimalPrefixedLengthParser(
 
 class PackedIntegerRuntimeLengthParser(
   val e: ElementRuntimeData,
-  signed: Boolean,
   packedSignCodes: PackedSignCodes,
   val lengthEv: Evaluatable[JLong],
   val lengthUnits: LengthUnits
-) extends PackedBinaryIntegerBaseParser(e, signed)
+) extends PackedBinaryIntegerBaseParser(e)
```

Comment:
I think you need to pass the signed boolean.

Reply:
Does using the signedness of the primitive type work? I think the only time xs:decimalSign should matter is with the primitive type is xs:decimal, which uses the `PackedDecimal*` parsers, not these `PackedInteger*` parsers. So if feels like we do need a signed boolean for `PackedDecimal*` parsers, but I think that's just an existing bug that could maybe be added in a separate PR, since this one is more about ranges of things.

Reply:
So this signed boolean was replaced by the primitives signedness...as we were just passing in true/false depending on whether the primitive was signed or unsigned...the PAckedDecimalParsers never passed in decimalSigned, so that wasn't remove...as @stevedlawrence said I think that's a separate bug

Reply:
Created https://issues.apache.org/jira/browse/DAFFODIL-2963 to track separate bug

Reply:
Also, https://github.com/apache/daffodil/pull/1391 modifies packed integers to use PrimType.fromNumber, which validates signedness according to the primitive, and the signed check in this parser goes away.